solarwinds/README.md

```diff
@@ -0,0 +1,101 @@
+## Overview
+
+Get alerts from SolarWinds Orion to aggregate and triage your alerts in a centralized location. 
+
+This integration works by subscribing Datadog to all of your SolarWinds alert notifications.
+
+## Setup
+
+### Create trigger actions
+
+To create a new trigger action in SolarWinds:
+
+1. Navigate to Alerts > Manage Alerts
+2. Select any alert and click “Edit Alert”, or create a new alert if you do not have any
+3. Navigate to Trigger Actions > Add Action
+4. Select “Send a GET or POST Request to a Web Server”
+5. Click “Configure Action”
+6. Fill in the Action Pane with the following details:
+
+        a. Name of Action: Send Alert to Datadog (or whatever you prefer)
+        b. URL: https://app.datadoghq.com/intake/webhook/solarwinds?api_key=<DATADOG_API_KEY>
+        c. Select: Use HTTP/S POST
+        d. Body to Post: Copy and paste from alert template below
+        e. Time of Day: Leave as is
+        f. Execution Settings: Leave as is
+
+7. Click “Add Action”
+8. Click the “Reset Actions” step and then repeat steps 4 - 7, using the Reset Action template instead of the Trigger Action template
```

Comment:
```suggestion
8. Click the “Reset Actions” step and then repeat steps 4 - 7, using the Reset Action template instead of the Trigger Action template.
```